tools/harness-thci/OpenThread.py

```diff
@@ -2963,13 +2963,19 @@ def setBbrDataset(self, SeqNumInc=False, SeqNum=None, MlrTimeout=None, ReRegDela
             False: fail to set BBR Dataset
         """
         assert not (SeqNumInc and SeqNum is not None), "Must not specify both SeqNumInc and SeqNum"
+
+        if MlrTimeout and MlrTimeout != self.bbrMlrTimeout or ReRegDelay and ReRegDelay != self.bbrReRegDelay:
```

Comment:
```suggestion
        if (MlrTimeout and MlrTimeout != self.bbrMlrTimeout) or (ReRegDelay and ReRegDelay != self.bbrReRegDelay):
```
nit: make the evaluation order clear since there are multiple `and` and `or`.

Reply:
Done.